heft-plugins/heft-jest-plugin/src/transformers/BuildTransformer.ts

```diff
@@ -0,0 +1,449 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
This is mostly a rename, right? What parts of this file changed?

Reply:
Rename + refactor. I added the cache key methods and the async methods as well, with the only functional difference being the use of async file I/O